psique/src/App/store/body/reducer.js

```diff
@@ -1,5 +1,5 @@
 const initialState = {
-    body: 'init'
+    body: 'WISC-registros'
```

Comment:
@Daniel0205 This line makes "Wisc-registros" the initial page - Change Home as the initial page